sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -100,6 +100,11 @@
       <version>4.1.49.Final</version> <!-- {x-version-update;io.netty:netty-transport-native-epoll;external_dependency} -->
       <classifier>linux-x86_64</classifier>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-kqueue</artifactId>
+      <version>4.1.49.Final</version> <!-- {x-version-update;io.netty:netty-transport-native-kqueue;external_dependency} -->
+    </dependency>
```

Comment:
This new dependency should also be added to the allowed list of dependencies. See line 193 for example. Also, this new external dependency has to be tracked in `eng/versioning/external_dependencies.txt`.

Reply:
Thanks for the help. Done the changes.